habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BehaveBddTestMojo.java

```diff
@@ -37,6 +38,20 @@ public class BehaveBddTestMojo extends AbstractHabushuMojo {
     @Parameter(property = "habushu.behaveOptions", required = false)
     protected String behaveOptions;
 
+    /**
+     * By default, format Behave test results in a JSON compatible with Cucumber Reports plugin
+     */
+    @Parameter(property = "habushu.outputCucumberStyleTestReports", required = false, defaultValue = "true")
+    protected boolean outputCucumberStyleTestReports;
+
+    /**
+     * By default, Behave marks Scenarios and Features with skipped steps as failures in its reports even if the
+     * other test steps themselves pass. To match Cucumber's default logic, this setting will prevent capturing skipped
+     * tests if there are no failures.
+     */
+    @Parameter(property = "habushu.dontFailBehaveTestsWithSkippedTestSteps", required = false, defaultValue = "true")
```

Comment:
I: Considering renaming this "omitSkippedTests" or inversing the logic and going with "showSkippedTests" - curious what others think.